mod.yaml

```diff
@@ -7,7 +7,8 @@ Metadata:
 RequiresMods:
 	modchooser: {DEV_VERSION}
 
-Folders:
+Packages:
+	~^Content/ra2
 	.
 	ra2:
 	ra2:bits
```

Comment:
You will want to define these after the content mix files if you want to be able to override any of the mix file contents.